### PR TITLE
Feature/11 new block copyright footer info

### PR DESCRIPTION
**Pull request by michal-adamowicz**
Add copyrights block with documentation.

Custom Features:
1. Editable 'Copyrights' text content
2. Automatic current year update
3. Option to display current blog title or custom text

Standard Features:
1. Colors
   -text
   -background
2. Spacing
   -margin
   -padding
3. Text Align

### CODE REVIEW

File: src/Core/Config.php [hunk 1]
```diff
@@ -32,9 +32,9 @@ public function addDependencies(): void
       'ajax' => admin_url('admin-ajax.php')
     ]);
 
-    if (! is_admin()) {
-      wp_deregister_style('common');
-    }
+    // if (! is_admin()) {
```
Comment (klapaucius4) [inline src/Core/Config.php:35]: we can remove commented lines if they are unnecessary

**Review by klapaucius4 — APPROVED** [review]
In my opinion is ok, only remove the commented lines please

**Review by coditive-michal-k — APPROVED** [review]
Sorry, I've made the review from my private account - my mistake. I set aprrove, but only remove the commented lines
